test/s3-benchmarks/src/main/java/software/amazon/awssdk/s3benchmarks/TransferManagerBenchmarkConfig.java

```diff
@@ -96,25 +105,46 @@ public Duration timeout() {
         return this.timeout;
     }
 
+    public Long memoryUsageInMb() {
+        return this.memoryUsageInMb;
+    }
+
+    public Long connectionAcquisitionTimeoutInSec() {
+        return this.connectionAcquisitionTimeoutInSec;
+    }
+
+    public boolean forceCrtHttpClient() {
+        return this.forceCrtHttpClient;
+    }
```

Comment:
This one confuses me. What's the use-case for doing 'java' but forcing the CRT HTTP client?

Reply:
The Multipart S3 Async Client can use Netty of CRT Http client. This forces the use of CRT Http client which we used to quickly compare the CRTS3AsyncCLient with the java based MultipartAsync Client configured with the CRT HTTP Client.